graphblas/tests/test_numpyops.py

```diff
@@ -61,6 +65,11 @@ def test_npunary():
         "INT64": {"reciprocal"},
         "FC64": {"ceil", "floor", "trunc"},
     }
+    if suitesparse and is_win:
+        # asin and asinh are known to be wrong in SuiteSparse:GraphBLAS
+        # due to limitation of MSVC with complex
+        blocklist["FC64"].update({"asin", "asinh"})
+        blocklist["FC32"] = {"asin", "asinh"}
```

Comment:
I don't know if we strictly need to exclude `asinh` given the input arguments we use, but `asinh` was called out in the documentation for not working.